actions/member.js

```diff
@@ -49,8 +51,14 @@ class List extends Action {
         } : {})
       },
       ...(limit !== -1 ? { offset, limit } : {}),
-      include: ['familyMembers'],
-      order: [['firstName', 'ASC'], ['lastName', 'ASC'], ['cardId', 'ASC']]
+      include: ['familyMembers']
+    }
+    if (order != null) {
```

Comment:
Why not check for truthy? `0` or `''` wouldn't make it sense

```suggestion
    if (order) {
```